templates/stamp/stamp.json

```diff
@@ -147,11 +147,59 @@
                 "description": "Number of application virtual machines behind the load balancer."
             }
         },
-        "branch": {
+        "customInstallerGithubAccountName": {
+            "type": "string",
+            "defaultValue": "microsoft",
+            "metadata": {
+                "description": "Name of the GitHub account where the custom installer lives"
+            }
+        },
+        "customInstallerGithubProjectName": {
+            "type": "string",
+            "defaultValue": "oxa-tools",
+            "metadata": {
+                "description": "Name of the GitHub project within the GitHub account where the custom installer lives"
+            }
+        },
+        "customInstallerGithubBranch": {
             "type": "string",
             "defaultValue": "master",
             "metadata": {
-                "description": "The GitHub branch from which scripts and other files will be downloaded"
+                "description": "The branch of the GitHub project from which custom installer scripts and other files will be downloaded"
+            }
+        },
+        "customInstallerGithubAccessToken": {
+            "type": "secureString",
+            "metadata": {
+                "description": "The personal access token to use when interacting with the private GitHub repository where the custom installer lives"
+            }
+        },
+        "githubAccountName": {
+            "type": "string",
+            "defaultValue": "microsoft",
+            "metadata": {
+                "description": "Name of the GitHub account where the Azure Arm Template lives"
+            }
+        },
+        "githubProjectName": {
+            "type": "string",
+            "defaultValue": "oxa-tools",
+            "metadata": {
+                "description": "Name of the GitHub project within the GitHub account where the Azure Arm Template lives"
+            }
+        },
+        "githubBranch": {
+            "type": "string",
+            "defaultValue": "master",
+            "metadata": {
+                "description": "The branch of the GitHub project from which scripts and other files will be downloaded"
+            }
+        },
+        "armTemplateRelativePath": {
+            "type": "string",
+            "defaultValue": "templates/stamp",
```

Comment:
re: armTemplateRelativePath - may be useful to introduce this as deployment type. We could use this to differentiate not only templates, but also edX configuration. Would allow us to differentiate certain config closely tied to the infrastructure, such as SMTP/SSL/etc.

I can look into this when I bolt on edx/oxa bootstrap.